src/App.scss

```diff
@@ -1,12 +1,12 @@
-@import "~@edx/edx-bootstrap/sass/edx/theme";
-@import "~bootstrap/scss/bootstrap";
+
+@import "~@edx/paragon/scss/edx/theme.scss";
 
 $fa-font-path: "~font-awesome/fonts";
 @import "~font-awesome/scss/font-awesome";
 
 $input-focus-box-shadow: $input-box-shadow; // hack to get upgrade to paragon 4.0.0 to work
-@import "~@edx/paragon/src/SearchField/SearchField";
-@import "~@edx/paragon/src/Collapsible/Collapsible";
+//@import "~@edx/paragon/src/SearchField/SearchField";
+//@import "~@edx/paragon/src/Collapsible/Collapsible";
```

Comment:
red diff best diff

Reply:
(to be clear, I want you to delete these rather than leave them as comments

Reply:
🤦‍♂  I never pushed the commit. Thanks for catching that.